### DISCUSSION TITLE
Bet365, can't acces it how much I try...

### DISCUSSION THREAD

**Discussion by JJjulle** [ultrafunkamsterdam/undetected-chromedriver#1881] [General]
Hi ,

I am not available to get pass the https://www.bet365.com/#/AC/B17/C20836572/D48/E972/F10/, where the odds information is...
I can get to the homepage but that is it.
I just want to understand if it possible or not, has anyone suceeded with this?
Would be very interested in understanding how it would be possible

**Comment by KebabLord** [comment]
maybe ssl is the problem:
```py
import undetected_chromedriver as uc
options = uc.ChromeOptions()
options.add_argument("--ignore-certificate-errors")
driver = uc.Chrome(options=options)
driver.get("https://www.bet365.com/")
```
